openshift/app.dc.yaml

```diff
@@ -0,0 +1,166 @@
+                  path: "${ROUTE_PATH}/api"
+                  port: 8080
+                  scheme: HTTP
+                initialDelaySeconds: 10
+                timeoutSeconds: 1
+                failureThreshold: 3
+              ports:
+                - containerPort: 8080
+                  protocol: TCP
+              readinessProbe:
+                httpGet:
+                  path: "${ROUTE_PATH}/api"
+                  port: 8080
+                  scheme: HTTP
+                initialDelaySeconds: 10
+                timeoutSeconds: 1
+                failureThreshold: 3
+              resources:
+                requests:
+                  cpu: "${CPU_REQUEST}"
+                  memory: "${MEMORY_REQUEST}"
+                limits:
+                  cpu: "${CPU_LIMIT}"
+                  memory: "${MEMORY_LIMIT}"
+              env:
+                - name: NODE_ENV
+                  value: production
+                - name: FRONTEND_BASEPATH
+                  value: "${ROUTE_PATH}"
+                - name: SERVER_BASEPATH
+                  value: "${ROUTE_PATH}"
+              envFrom:
+                - configMapRef:
+                    name: "${APP_NAME}-frontend-config"
+                - configMapRef:
+                    name: "${APP_NAME}-server-config"
+          restartPolicy: Always
+          terminationGracePeriodSeconds: 30
+      test: false
+      triggers:
+        - type: ConfigChange
+  - apiVersion: v1
+    kind: Service
+    metadata:
+      name: "${APP_NAME}-app-${JOB_NAME}"
+    spec:
+      ports:
+        - name: 8080-tcp
+          port: 8080
+          protocol: TCP
+          targetPort: 8080
+      selector:
+        app: "${APP_NAME}-${JOB_NAME}"
+        deploymentconfig: "${APP_NAME}-app-${JOB_NAME}"
+        role: app
+      sessionAffinity: None
+  - apiVersion: v1
+    kind: Route
+    metadata:
+      name: "${APP_NAME}-app-${JOB_NAME}"
+    spec:
+      host: "${ROUTE_HOST}"
+      path: "${ROUTE_PATH}"
+      port:
+        targetPort: 8080-tcp
+      tls:
+        insecureEdgeTerminationPolicy: Redirect
+        termination: edge
+      to:
+        kind: Service
+        name: "${APP_NAME}-app-${JOB_NAME}"
+        weight: 100
+      wildcardPolicy: None
+parameters:
+  - name: APP_NAME
+    description: Application name
+    displayName: Application name
+    required: true
+  - name: ROUTE_HOST
+    description: The host the route will use to expose service outside cluster
+    displayName: Route host
+    required: true
+  - name: ROUTE_PATH
+    description: Configure the route path (ex. /pr-5 or /app), also used for FRONTEND_BASEPATH
+    displayName: Route path
+    required: true
+  - name: JOB_NAME
+    description: Job identifier (i.e. 'pr-5' OR 'master')
+    displayName: Job Branch Name
+    required: true
+  - name: IMAGE_REGISTRY
+    description: The docker image registry
+    displayName: Docker Image Registry
+    required: true
+    value: ghcr.io/bcgov
```

Comment:
This should be re-templated to `image: "${IMAGE_REGISTRY}/${IMAGE_NAMESPACE}/${IMAGE_NAME}:${IMAGE_TAG}"` or similar - lumping image registry with both the registry and the namespace as a default is bad practice.